roles/install/tasks/github.yml

```diff
@@ -14,8 +14,36 @@
   ansible.builtin.set_fact:
     _community_sops_install_effective_sops_version: '{{ sops_version }}'
 
+# This method asks GitHub for the latest release, which depends on the release to be
+# correctly marked as "latest" in the GitHub UI. Fortunately this is not as aggressively
+# rate-limited as the API (used in the fallback).
 - name: Determine latest version
-  when: sops_version == 'latest'
+  when: sops_version == 'latest' and true
+  block:
+    - name: Fetch the latest release from GitHub
+      ansible.builtin.uri:
+        url: https://github.com/mozilla/sops/releases/latest/
+        follow_redirects: false
+        status_code:
+          - 302
+          - 307
+      register: _community_sops_install_github_latest_release
+      delegate_to: localhost
+      run_once: true
+
+    - name: Determine the latest release
+      ansible.builtin.set_fact:
+        _community_sops_install_effective_sops_version: >-
+          {{
+            _community_sops_install_github_latest_release.location
```

Comment:
Relying on undocumented HTTP redirect seems quite fragile and might not avoid preleases and drafts as the original one does.

Have you thought of using [automatically provided `GITHUB_TOKEN` secret](https://docs.github.com/en/actions/security-guides/automatic-token-authentication) and passing that to the role, increasing hourly limit from 60 to 1000?

Reply:
I don't think the redirect is really undocumented. https://docs.github.com/en/repositories/releasing-projects-on-github/linking-to-releases explicitly mentions /releases/latest (though it does not mention it is a redirect).